src/main/java/com/fvv/bookstore/bean/Laptop.java

```diff
@@ -14,32 +14,23 @@
 
 	private static final long serialVersionUID = 1L;
 	private Integer ramSize;
-	private String processor;
-	private Double screenSize;
 	private Double hdSize;
 	
 	/**
 	 * Class constructor specifying the attributes and with super from the Class Hardware
 	 * @param brand of the Laptop
-	 * @param model of the Laptop
-	 * @param color of the Laptop
 	 * @param price of the Laptop
 	 * @param warranty of the Laptop
 	 * @param ramSize of the Laptop
-	 * @param processor of the Laptop
-	 * @param screenSize of the Laptop
 	 * @param hdSize of the Laptop
 	 */
-	public Laptop(final String brand, final String model, final String color, final Double price, 
-			final Integer warranty, final Integer ramSize, final String processor, final Double 
-			screenSize, final Double hdSize) {
-		super(brand, model, color, price, warranty);
+	public Laptop(final String brand, final Double price, final Integer warranty, 
```

Comment:
your laptop does not have a processor, how are them turn on?

Reply:
ok